spec/models/comment_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'associations' do
+    it { should belong_to(:author).class_name('User') }
+    it { should belong_to(:post) }
+  end
+
+  describe 'update_comments_counter' do
+    let!(:user) { User.create!(name: 'John Doe', post_counter: 0) }
+    let!(:post) { Post.create!(title: 'My First Post', author: user, comments_counter: 0, likes_counter: 0) }
+    it 'increments the post comment counter after saving' do
+      expect { comment.save }.to change { post.reload.comments_counter }.from(0).to(1)
+    end
+  end
```

Comment:
- [ ] @SSEKPIUS kindly check on this test,it fails when I run it, below is a screenshot of this and other tests that are failing, kindly ensure all tests are passing

![image](https://user-images.githubusercontent.com/61408860/224469108-ca4d8f98-ab0a-4cc0-bdcb-ec5e2e50946a.png)